cardano_node_tests/tests/test_cli.py

```diff
@@ -207,6 +207,57 @@ def test_tx_view(self, cluster: clusterlib.ClusterLib):
         elif "witnesses:" not in tx:
             assert tx == tx_body
 
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    def test_query_tip(self, cluster: clusterlib.ClusterLib):
+        """Test `query tip`."""
+        tip_out = json.loads(
+            cluster.cli(
+                [
+                    "query",
+                    "tip",
+                    *cluster.magic_args,
+                ]
+            ).stdout.decode("utf-8")
+        )
+
+        expected_out = {
+            "block",
+            "epoch",
+            "era",
+            "hash",
+            "slot",
+            "slotInEpoch",
+            "slotsToEpochEnd",
+            "syncProgress",
+        }
+
+        # check that 'query tip' is returning the expected fields
+        assert (
+            set(tip_out.keys()) == expected_out
+        ), "The output fields of 'query tip' are not the expected"
```

Comment:
I don't think that "are not the expected" is grammatically correct.
Could be "... not the expected ones", or even better "Unexpected fields in `query tip` output"